### DISCUSSION TITLE
离线下载能否同时转存和做种

### DISCUSSION THREAD

**Discussion by bsdayo** [cloudreve/cloudreve#1369] [General]
最近使用 docker-compose 搭建的 cloudreve，在用离线下载功能下载种子的时候发现 cloudreve 会在做种结束后再进行转存，这样就造成了时间浪费

文档里建议的做法是直接在 aria2.conf 里把 `seed-time` 改成 0，直接禁用做种，但是本着分享精神打算把做种时间改为一天，一星期或者更久，这样就会导致在 cloudreve 里需要等待相同的时间才能使用下载的文件

有没有可能可以做到同时转存和做种，例如本地存储就通过软链接等方式，云端存储就增加选项是否转存后删除源文件

知识比较匮乏，如果有没有考虑到的问题还请指出和谅解（

**Comment by chivehao** [comment]
确实，希望能支持做种同时转存，毕竟你的下载也是因为别人的做种才有的，礼尚往来才是长久之道。
